projects/hslayers/src/components/add-data/common/common-file.service.ts

```diff
@@ -302,7 +302,9 @@ export class HsAddDataCommonFileService {
         layerName,
         endpoint.user
       );
-      if (['STARTED', 'PENDING', 'SUCCESS'].includes(descriptor.wms.status)) {
```

Comment:
Shouldn't we keep the previous statuses for ongoing processing? For the sake of backwards compatibility at least.

Reply:
Didn't consider it! Will add them back for the backwards compatibility.